ui_framework/doc_site/src/views/accessibility/keyboard_accessible.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+  KuiKeyboardAccessible,
+} from '../../../../components';
+
+// For custom components, we just need to make sure they delegate props to their rendered root
+// element, e.g. onClick, tabIndex, and role.
+const CustomComponent = ({
+  children,
+  ...rest,
+}) => (
+  <div {...rest}>
+    {children}
+  </div>
+);
+
+export default () => (
+  <div>
+    <KuiKeyboardAccessible>
+      <div onClick={() => window.alert('Div clicked')}>
+        Click this div
+      </div>
+    </KuiKeyboardAccessible>
+
+    <KuiKeyboardAccessible>
+      <a
+        className="kuiLink"
+        onClick={() => window.alert('Anchor tag clicked')}
+      >
+        Click this anchor tag
+      </a>
+    </KuiKeyboardAccessible>
+
+    <KuiKeyboardAccessible>
+      <CustomComponent onClick={() => window.alert('Custom component clicked')}>
+        Click this custom component
+      </CustomComponent>
+    </KuiKeyboardAccessible>
+
+    <KuiKeyboardAccessible>
+      <div onClick={() => window.alert('Div clicked')}>
+        Click this div, which itself contains another acessible element:&nbsp;
```

Comment:
I'm not sure if it should trigger both or not. I was mostly commenting because it _did_ trigger both when I played around with this and I wasn't sure if it was expected or not, so my comment was just about making intention/expectation clear in the "Click this div" text.